### PR TITLE
Travis testing

### PR DESCRIPTION
**Pull request by kehphin**
Accidentally removed .travis.yml - This PR puts it back so we have CI testing.
We also deprecate Node 4 in our testing, requiring a minimum of Node 6.

PR Prime: @dfreeman

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "test:all": "ember try:each"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "^4.5 || 6.* || >= 7.*"
```
Comment (dfreeman) [inline package.json]: I think what we had here is fine, since Node 4 is deprecated. We should just update Travis to run with Node 6 instead

Comment (kehphin) [reply package.json]: Ok, wasn't sure if we wanted to support users of node 4

**Comment by dfreeman** [comment]
Thanks @kehphin!
